base/util_ce.go

```diff
@@ -56,3 +56,8 @@ func JSONEncoder(w io.Writer) JSONEncoderI {
 func JSONEncoderCanonical(w io.Writer) JSONEncoderI {
 	return json.NewEncoder(w)
 }
+
+// JSONMarshalIndent is like JSONMarshal but applies Indent to format the output.
+func JSONMarshalIndent(v interface{}, prefix, indent string) ([]byte, error) {
```

Comment:
This file is CE only - there's no MarshalIndent equivalent added to the EE file, so this fails to compile for EE.